### PR TITLE
Initial commit for marketing

### PR DESCRIPTION
**Pull request by elfgoh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,56 @@ Highly inspired by @winston's [Conference Planning Playbook](https://github.com/
...
+- D-day postings for each talk: start and end
+- Sponsor booths
+- Food
+- After party
+- Post event Survey
+- Video uploads for each talk
+- Video playlist link
+
+#### Resources
+- Social media accounts: [FB](https://www.facebook.com/GeekcampSG/), [Twitter](https://twitter.com/geekcamp), [Instagram](https://www.instagram.com/geekcampsg)
```
Comment (perwyl) [inline README.md:92]: YouTube 
Slack

File: README.md [hunk 2]
```diff
@@ -44,7 +44,56 @@ Highly inspired by @winston's [Conference Planning Playbook](https://github.com/
...
+##### Prepare messaging templates for event phases
+- Recruitment
+- Team meeting pictures
+- Sponsorship deck
+- CFP + conference date
+- CFP closing soon / CFP extension
+- Confirmed schedule
+- Registration
+- Sponsor shoutouts
+- Reminder of event at D01
```
Comment (alanswanht) [inline README.md]: What's `D01`? Is it supposed to be `D-1`?

File: README.md [hunk 3]
```diff
@@ -44,7 +44,56 @@ Highly inspired by @winston's [Conference Planning Playbook](https://github.com/
...
+##### Queries
+- Ensure that queries sent via email and on social media are answered promptly
+
+##### Prepare messaging templates for event phases
+- Recruitment
+- Team meeting pictures
+- Sponsorship deck
+- CFP + conference date
+- CFP closing soon / CFP extension
+- Confirmed schedule
```
Comment (alanswanht) [inline README.md:77]: - Speakers promotion
- Collect speakers' social media handles

File: README.md [hunk 4]
```diff
@@ -44,7 +44,56 @@ Highly inspired by @winston's [Conference Planning Playbook](https://github.com/
...
+- Sponsor shoutouts
+- Reminder of event at D01
+- D-day postings for each talk: start and end
+- Sponsor booths
+- Food
+- After party
+- Post event Survey
+- Video uploads for each talk
+- Video playlist link
+
```
Comment (alanswanht) [inline README.md:90]: - Post-event promotion (including video links above)

Comment (elfgoh) [reply README.md:90]: @alanswanht I dont quite understand what you are referring to. Can you please give an example?

Comment (alanswanht) [reply README.md:90]: @elfgoh  Eg: The tweets and FB posts we have just scheduled for the videos and playlist.

**Pull request merged**